crawl-ref/source/item-name.cc

```diff
@@ -1108,32 +1125,53 @@ string sub_type_string(const item_def &item, bool known)
     case OBJ_POTIONS: return potion_type_name(sub_type);
     case OBJ_BOOKS:
     {
-        if (sub_type == BOOK_MANUAL)
+        switch (sub_type)
         {
+        case BOOK_MANUAL:
+            {
             if (!known)
                 return "manual";
             string bookname = "manual of ";
             bookname += skill_name(static_cast<skill_type>(item.plus));
             return bookname;
-        }
-        else if (sub_type == BOOK_NECRONOMICON)
+            }
+        case BOOK_NECRONOMICON:
             return "Necronomicon";
-        else if (sub_type == BOOK_GRAND_GRIMOIRE)
+        case BOOK_GRAND_GRIMOIRE:
             return "Grand Grimoire";
 #if TAG_MAJOR_VERSION == 34
-        else if (sub_type == BOOK_BUGGY_DESTRUCTION)
+        case BOOK_BUGGY_DESTRUCTION:
             return "tome of obsoleteness";
 #endif
-        else if (sub_type == BOOK_YOUNG_POISONERS)
+        case BOOK_EVERBURNING:
+            return "Everburning Encyclopedia";
+        case BOOK_OZOCUBU:
+            return "Ozocubu's Autobio";
+        case BOOK_YOUNG_POISONERS:
             return "Young Poisoner's Handbook";
-        else if (sub_type == BOOK_FEN)
+        case BOOK_FEN:
             return "Fen Folio";
+        case BOOK_MOON:
+            return "Lunar Lorehold";
+        case BOOK_NEARBY:
+            return "Inescapable Indices";
```

Comment:
```suggestion
            return "Zohar";
```

Less confident about that one. More bike shedding is ongoing.

Reply:
Will hold off on this for now.